pkg/rbd/controllerserver.go

```diff
@@ -360,7 +352,8 @@ func (cs *ControllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateS
 
 	rbdSnap.VolName = rbdVolume.VolName
 	rbdSnap.SnapName = snapName
-	snapshotID := "csi-rbd-" + rbdVolume.VolName + "-snap-" + uniqueID
+	//Generating Snapshot ID
+	snapshotID := "csi-rbd-snap-" + snapName
```

Comment:
Isn't it useful to keep the source image name in the `snapshotID`?

Reply:
updated code to use source image name